cmd/agent/app/jmx.go

```diff
@@ -89,6 +92,7 @@ var (
 
 	cliSelectedChecks = []string{}
 	jmxLogLevel       string
+    saveFlare         bool
```

Comment:
The code uses tabs and not spaces. See gofmt formater: https://golang.org/cmd/gofmt/.
 > Gofmt formats Go programs. It uses tabs for indentation and blanks for alignment. Alignment assumes that an editor is using a fixed-width font.

It would also fixe the failed test `ci/circleci: unit_tests`